### ISSUE TITLE
Unresolved reference: R

### ISSUE THREAD

**Issue by madroidmaq** [ponnamkarthik/FlutterToast#288]
Use a higher version of gradle (such as: 6.6.1), the following error will occur：

```shell
e: /usr/local/Caskroom/flutter/2.0.2/flutter/.pub-cache/hosted/pub.flutter-io.cn/fluttertoast-8.0.1-nullsafety.0/android/src/main/kotlin/io/github/ponnamkarthik/toast/fluttertoast/MethodCallHandlerImpl.kt: (45, 84): Unresolved reference: R
e: /usr/local/Caskroom/flutter/2.0.2/flutter/.pub-cache/hosted/pub.flutter-io.cn/fluttertoast-8.0.1-nullsafety.0/android/src/main/kotlin/io/github/ponnamkarthik/toast/fluttertoast/MethodCallHandlerImpl.kt: (70, 68): Unresolved reference: R
e: /usr/local/Caskroom/flutter/2.0.2/flutter/.pub-cache/hosted/pub.flutter-io.cn/fluttertoast-8.0.1-nullsafety.0/android/src/main/kotlin/io/github/ponnamkarthik/toast/fluttertoast/MethodCallHandlerImpl.kt: (85, 64): Unresolved reference: R
```

**Comment by jagomf** [comment]
Getting same error:

![image](https://user-images.githubusercontent.com/4032522/112987301-db7a9b00-9162-11eb-94ea-fc748fdce453.png)

**Comment by jagomf** [comment]
Same issue as #285

**Comment by ctedgar** [comment]
> same issue here can you please resolve and update package.

+1

**Comment by cor277** [comment]
+1

**Comment by ponnamkarthik** [comment]
I was unable to reproduce the issue can one you be able to provide the project zip with the bug

**Comment by GabrielDiBastiani** [comment]
> I was unable to reproduce the issue can one you be able to provide the project zip with the bug

@ponnamkarthik  here is the code to reproduce the issue: https://github.com/GabrielDiBastiani/toast-test

The error is related with `compileSdkVersion 29` in build.gradle, as you can see in this workaround https://github.com/ponnamkarthik/FlutterToast/issues/284#issuecomment-802577989

**Comment by ponnamkarthik** [comment]
FIxed in `8.0.4`
